cluster-topology.md

```diff
@@ -0,0 +1,151 @@
+# Use Cluster API to change the Cluster Topology
+
+[In the previous walk-through](./your-first-cluster.md) we demonstrated how to create a cluster using a standard declarative configuration. Now we will show how to leverage this declarative configuration to update cluster topology with ease.
+
+## What is Cluster Topology?
+
+We define a cluster topology as the set of configurations that describe your cluster: a few examples are the number of control plane and worker nodes; the type of Machine hardware that underlies various nodes; regional distribution across nodes or node pools. You may also see this described as "cluster shape" elsewhere.
+
+## Scale out control plane nodes
+
+A common Kubernetes cluster maintenance activity is scaling out (or in) the number of control plane nodes in response to cluster activity. Because Cluster API configuration interfaces are in fact Kubernetes resources, there are a lot of ways to do this. We'll demonstrate using a variety of methods.
+
+### Scale from 1 to 3 control plane nodes via kubectl apply
+
+Because we are leveraging the Kubernetes declarative model, we can simply refer to a desired configuration specification and rely upon Cluster API to evaluate what's different between the two.
+
+Assuming that your `docker-cluster-one` is in its original configuration, you should have one control plane node:
+
+```sh
+$ kubectl --kubeconfig cluster-one.kubeconfig get nodes
+NAME                                             STATUS   ROLES           AGE   VERSION
+docker-cluster-one-md-0-nrh7k-75ddc4778f-vlph9   Ready    <none>          24m   v1.24.6
+docker-cluster-one-xbqb2-tlmpb                   Ready    control-plane   24m   v1.24.6
+```
+
+Let's use the idempotent model to submit a modified configuration of our `docker-cluster-one` Cluster with 3 control plane replicas instead of 1. We've provided [a reference yaml of this updated configuration in this repo](yamls/clusters/docker-cluster-one-3-control-plane-replicas.yaml).
+
+If you're on masOS or Linux you can diff this modified spec from the original spec used to create the `docker-cluster-one` Cluster:
```

Comment:
I wonder if we should just simplify this sentence to something like. The diff between those two files is the following:

I wonder if it get's annoying if we have to repeat this sentence all over and I don't expect folks to run the diff command manually on top of the diff that we already provide

Reply:
Done